test/client/fixtures/sandbox/code-instrumentation/getters-test.js

```diff
@@ -481,3 +481,37 @@ test('we should clean up hammerhead script and style for all elements(GH-1079)',
     ok(checkProperty(innerHTML));
     ok(checkProperty(outerHTML));
 });
+
+test('we should clean up html and remove extra namespaces from svg (GH-1083)', function () {
+    var svg       = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
```

Comment:
You use reqular expession so need to add test cases for all variants:
`innerHtml` doesn't contains `svg` tag
`innerHtml` contains two `svg` tags
`svg` tag without `NS` attribute
`svg` tag with two `NS` attribute